bundles/org.openhab.binding.freeathomesystem/README.md

```diff
@@ -0,0 +1,169 @@
+```
+
+## Examples for .things
+
+Things are all discovered automatically and visible on the openHAB dashboard after pushing the scan button
+
+If you really like to manually configure a thing:
+
+```java
+Bridge freeathomesystem:bridge:mysysap [ ipaddress="...", username="...", password="..." ]
+{
+  Thing free-at-home-device    ABB700997F0A
+}
+```
+
+The only parameter needed to creta a Thing is the free@home device ID, which you can find as sticker on the device.
+The creation of the openHAB channels to operate the free@home device is happening automaticlly based on the device features detected on-line.
+
+## Examples for .items
+
+Sample for the free@home thermostat device
+
+```java
+Number Livingroom_MeasuredTemperature                     "Measured Temperature"            <temperature>  (Livingroom)  ["Measurement"]  { channel="freeathomesystem:thermostat:0836e63805:ABB700CF0FB0_0:thermostatMeasuredTemperature" }
+Number LivingRoom_Thermostat_SetpointTemperature          "Setpoint Temperature"            <temperature>  (Livingroom)  ["Setpoint"]     { channel="freeathomesystem:thermostat:0836e63805:ABB700CF0FB0_0:thermostatSetpointTemperature" }
+Switch LivingRoom_Thermostat_ThermostatActivationOnOff    "Thermostat Activation"           <switch>       (Livingroom)  ["Setpoint"]     { channel="freeathomesystem:thermostat:0836e63805:ABB700CF0FB0_0:thermostatOnoffSwitch" }
+Switch LivingRoom_Thermostat_MainEcoOnOff                 "Thermostat Eco Activation"       <switch>       (Livingroom)  ["Setpoint"]     { channel="freeathomesystem:thermostat:0836e63805:ABB700CF0FB0_0:thermostatEcoSwitch" }
+Number LivingRoom_ThermostatHeatingActive                 "Thermostat Heating Active"       <temperature>  (Livingroom)  ["Status"]       { channel="freeathomesystem:thermostat:0836e63805:ABB700CF0FB0_0:thermostatHeatingActive" }
+Number LivingRoom_ThermostatHeatingDemand                 "Thermostat Heating Demand"       <temperature>  (Livingroom)  ["Status"]       { channel="freeathomesystem:thermostat:0836e63805:ABB700CF0FB0_0:thermostatHeatingDemand" }
+```
+
+Sample for the free@home sensor/actuator 1/1
+```
+Switch SwitchMoodlight_Livingroom                         "Moodlight Livingroom"     <switch>  (Livingroom)  ["Light"]   { channel="freeathomesystem:actuator:0836e63805:ABB700D85E0A_0:switchChannel }
+```
+
+Sample for the free@home sensor/actuator 2/2
+```
+Switch Switch_Livingroom_Double_1                         "Moodlight Livingroom"     <switch>  (Livingroom)  ["Light"]   { channel="freeathomesystem:actuator:0836e63805:ABB700887F0A_0:switchChannel }
+Switch Switch_Livingroom_Double_2                         "Moodlight Livingroom"     <switch>  (Livingroom)  ["Light"]   { channel="freeathomesystem:actuator:0836e63805:ABB700887F0A_1:switchChannel }
+```
+
+## Debugging and Tracing
+If you want to see what's going on in the binding, switch the log level to DEBUG in the Karaf console
```

Comment:
Don't think you should mention the debugging and logging at this place. Debugging and logging has it's own documantation page and this might be duplicate/wrong.

Reply:
I will solve